pytorch/training/docker/1.12/py3/example/Dockerfile.gpu.example.os_scan_allowlist.json

```diff
@@ -56,5 +56,37 @@
       "status": "ACTIVE",
       "title": "CVE-2022-45907 - torch"
     }
-  ]
+  ],
```

Comment:
Recommend adding the same "reason_for_ignore" to the 2 blocks above as they are listed as critical, though they are patched.